keyboards/czc/dwarf/keymaps/via/keymap.c

```diff
@@ -0,0 +1,55 @@
+    ),
+    [1] = LAYOUT_tkl_ansi(
+        QK_BOOT,          KC_MYCM, KC_WHOM, KC_MAIL, KC_MSEL, KC_MRWD, KC_MFFD, KC_MPLY, KC_MSTP, KC_VOLU, KC_VOLD, KC_MUTE, KC_CALC,    _______, RGB_MOD, RGB_HUI,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,    _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,    _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
+        _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,             RGB_VAI,
+        _______, GU_TOGG, _______,                            _______,                            _______, _______, _______, _______,    RGB_SPD, RGB_VAD, RGB_SPI
+    ),
+    [2] = LAYOUT_tkl_ansi(
+        _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,    _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,    _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,    _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
+        _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,             _______,
+        _______, _______, _______,                            _______,                            _______, _______, _______, _______,    _______, _______, _______
+    ),
+    [3] = LAYOUT_tkl_ansi(
+        _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,    _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,    _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,    _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
+        _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,             _______,
+        _______, _______, _______,                            _______,                            _______, _______, _______, _______,    _______, _______, _______
+    )
```

Comment:
extra blank layers should not be needed anymore on VIA builds. Build system should take care of it, leaving keymap files cleaner
```suggestion
    )
```